hoodie-client/src/test/java/com/uber/hoodie/index/bloom/TestHoodieGlobalBloomIndex.java

```diff
@@ -211,10 +211,10 @@ public void testExplodeRecordRDDWithFileComparisons() {
         t -> t._2()._2().getRecordKey(), Collectors.mapping(t -> t._2()._1().split("#")[0], Collectors.toList())));
 
     assertEquals(4, recordKeyToFileComps.size());
-    assertEquals(Arrays.asList("f4", "f1", "f3"), recordKeyToFileComps.get("002"));
-    assertEquals(Arrays.asList("f4", "f1", "f3"), recordKeyToFileComps.get("003"));
-    assertEquals(Arrays.asList("f4", "f1"), recordKeyToFileComps.get("004"));
-    assertEquals(Arrays.asList("f4", "f1"), recordKeyToFileComps.get("005"));
+    assertEquals(new HashSet<>(Arrays.asList("f4", "f1", "f3")), new HashSet<>(recordKeyToFileComps.get("002")));
```

Comment:
Would it be easy to cover range pruning on/off cases here ?

Reply:
that is purely a performance related artifact right? I ll look at improving coverage holistically, down the line